### DISCUSSION TITLE
Watched Filter for Actor Overview

### DISCUSSION THREAD

**Discussion by Treveo** [trakt/showly#873] [Ideas / Feature Requests]
**Feature Request:**

Add a 'Watched' filter option in the actor overview, allowing users to view only the movies and series they have already watched. Currently, users can navigate to an actor’s profile and see a list of all movies and series they’ve been in, but there's no way to filter this list based on what has already been watched.

This feature would be especially useful for moments when users think, _“This actor looks familiar, but I can’t remember where I’ve seen them before.”_ By applying the 'Watched' filter, users can quickly check which movies or series they’ve already seen featuring that actor. This would help them recall previous viewings and make connections more easily.

**Benefits:**

- Allows users to quickly see their viewing history for a specific actor.
- Helps users identify where they recognize an actor from.
- Improves organization and clarity within the app.
- Enhances the user experience by making navigation more efficient and intuitive.

**Comment by michaldrabik** [comment]
Thanks! I'm gonna add this to backlog, I guess its easy enough to implement and makes sense

**Reply by Treveo** [reply]
Thank you! 

P.S.: I love the new UI with the floating "bottom" buttons. Keep up the good work!

**Comment by Treveo** [comment]
![Screenshot_20250525_141649_Showly](https://github.com/user-attachments/assets/787612ce-4963-4996-916f-c2e3ce9ecbc0)

Thanks for this filter. Tho I don't think it is working 100%. This one is Tom Hanks and I have "Collection" filter on. It looks like that the last 2 ones at bottom aren't correct. I haven't seen those 2 and they are not marked by me at all.

**Reply by michaldrabik** [reply]
Interesting, are they in your Watchlist maybe?

**Reply by Treveo** [reply]
> Interesting, are they in your Watchlist maybe?

Not at all. I have nothing with those movies

**Reply by michaldrabik** [reply]
It was indeed a bug with filtering there. Found it already.
